homeassistant/components/camera/webrtc.py

```diff
@@ -29,6 +30,26 @@
 )
 
 
+@dataclass(kw_only=True)
+class WebRTCClientConfiguration:
```

Comment:
We're moving this back and forth. This needs some explanation what the plan is.

Reply:
As discussed separately, this doesn't need to be a until since only camera platforms are using it. 👍